spec/beaker/host/unix/pkg_spec.rb

```diff
@@ -145,6 +145,48 @@ def exec
 
     end
 
+    context "install_package_with_rpm" do
+
+      it "accepts a package as a single argument" do
+        @opts = {'platform' => 'el-is-me'}
+        pkg = 'redhat_package'
+        expect( Beaker::Command ).to receive(:new).with("rpm  -ivh #{pkg} ", [], {:prepend_cmds=>nil, :cmdexe=>false}).and_return('')
+        expect( instance ).to receive(:exec).with('', {}).and_return(generate_result("hello", {:exit_code => 0}))
+        expect( instance.install_package_with_rpm(pkg) ).to be == "hello"
+      end
+
+      it "accepts a package and additional options" do
+        @opts = {'platform' => 'el-is-me'}
+        pkg = 'redhat_package'
+        cmdline_args = 'foo'
+        expect( Beaker::Command ).to receive(:new).with("rpm #{cmdline_args} -ivh #{pkg} ", [], {:prepend_cmds=>nil, :cmdexe=>false}).and_return('')
+        expect( instance ).to receive(:exec).with('', {}).and_return(generate_result("hello", {:exit_code => 0}))
+        expect( instance.install_package_with_rpm(pkg, cmdline_args) ).to be == "hello"
```

Comment:
@kevpl This test shows that `install_package_with_rpm` can be called without an options hash.